src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.GetProcessMemoryInfo.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    [StructLayout(LayoutKind.Sequential)]
+    internal struct ProcessMemoryCounters
+    {
+        public uint cb;
+        public uint PageFaultCount;
+        public ulong PeakWorkingSetSize;
```

Comment:
This structure definition is not correct. The C/C++ version has `SIZE_T`. `SIZE_T` maps to `UIntPtr` in C#.